helm/broker-app/templates/deploymentconfig.yaml

```diff
@@ -27,6 +27,18 @@ spec:
       labels:
         app: {{ .Values.global.name }}
         deploymentconfig: {{ .Values.global.name }}
+    annotations:
+        vault.hashicorp.com/agent-inject: 'true'
+        vault.hashicorp.com/agent-inject-secret-broker-token: a03c8f-{{ .Values.global.envlable }}/{{ .Values.global.envtag }}-token
+        vault.hashicorp.com/agent-inject-template-broker-token: |
+          {{`{{- with secret `}}"a03c8f-{{ .Values.global.envlable }}/{{ .Values.global.envtag }}-token" {{` }}
```

Comment:
License plate is a quasi-secret. Extract from: .Release.Namespace

https://helm.sh/docs/chart_template_guide/builtin_objects/